<launch>
  <node pkg="rosserial_python" type="serial_node.py" name="rosserial_lipo" output="screen">
  <param name="~port" value="/dev/ttyACM0" />
  </node>
</launch>
